pkg/project/project.go

```diff
@@ -1,13 +1,18 @@
 package project
 
 var (
-	description        = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
-	gitSHA             = "n/a"
-	name        string = "aws-operator"
-	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "n/a"
+	bundleVersion        = "6.4.0"
+	description          = "The aws-operator handles Kubernetes clusters running on a Kubernetes cluster inside of AWS."
+	gitSHA               = "n/a"
+	name          string = "aws-operator"
+	source        string = "https://github.com/giantswarm/aws-operator"
+	version              = "n/a"
 )
 
+func BundleVersion() string {
```

Comment:
Can we simply call it `Version`?

Reply:
We already have `Version` which is the git version detected at build time.

I would say we start with this, and iterates from there.